interface-definitions/container.xml.in

```diff
@@ -332,9 +332,27 @@
           </leafNode>
         </children>
       </tagNode>
-      <leafNode name="registry">
+      <tagNode name="registry">
         <properties>
           <help>Registry Name</help>
+        </properties>
+        <children>
+          #include <include/generic-disable-node.xml.i>
+          <leafNode name="username">
+            <properties>
+              <help>User name for authentication</help>
+            </properties>
+          </leafNode>
+          <leafNode name="password">
+            <properties>
+              <help>Password for authentication</help>
```

Comment:
Please add a regex for matching passwords, `<regex>[[:ascii:]]{1,128}</regex>` 1-128 characters should probably be sufficient.